### PR TITLE
Deprecate connection strings for IBM-cloud-databases

### PR DESCRIPTION
**Pull request by lornakelly**
Add deprecation note for `connectionstrings` in favour of `ibm_database_connection`

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:
```
=== RUN   TestAccIBMDatabaseDataSource_basic
=== PAUSE TestAccIBMDatabaseDataSource_basic
=== CONT  TestAccIBMDatabaseDataSource_basic
--- PASS: TestAccIBMDatabaseDataSource_basic (641.03s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/database        642.116s
```

### CODE REVIEW

File: website/docs/d/database.html.markdown [hunk 1]
```diff
@@ -89,4 +89,4 @@ In addition to all argument references list, you can access the following attrib
 
 
 **Note**
-The provider only exports the admin user ID and associated connection string. It does not export any user IDs that are configured for the instance in addition.
```
Comment (omaraibrahim) [inline website/docs/d/database.html.markdown:92]: Did you mean to make a change here?

Comment (lornakelly) [reply website/docs/d/database.html.markdown]: I did not but cant see what the change actually is!

Comment (lornakelly) [reply website/docs/d/database.html.markdown]: Got it, was new line change

**Comment by obai-1** [comment]
@lornakelly we should also deprecate `connectionstrings` in `ibm/service/database/resource_ibm_database.go`

**Pull request merged**